lib/sycamore/sycamore/writer.py

```diff
@@ -266,17 +268,17 @@ def weaviate(
             WeaviateClientParams,
             WeaviateWriterTargetParams,
         )
-        from sycamore.connectors.weaviate.weaviate_writer import CollectionConfigCreate
+        from weaviate.collections.classes.config import _CollectionConfigCreate
```

Comment:
`# Importing _ prefixed stuff is fairly common for users of the weaviate codebase`

Reply:
Done.